metabackup/main.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/pingcap/tidb-cloud-backup/pkg"
+)
+
+const (
+	basicPath = "metadata_backup"
+)
+
+var (
+	cloud        string
+	region       string
+	bucket       string
+	cephEndpoint string
+	etcdEndpoint string
+)
+
+func init() {
+	flag.StringVar(&cloud, "cloud", "ceph", "Cloud storage to use")
+	flag.StringVar(&region, "region", "", "The region to send requests to.")
+	flag.StringVar(&bucket, "bucket", "tidb-meta-backup", "Name of bucket")
+	flag.StringVar(&cephEndpoint, "cephEndpoint", "", "Endpoint of Ceph object store")
+	flag.StringVar(&etcdEndpoint, "etcdEndpoint", "", "Endpoint of Etcd object store")
+	flag.Parse()
+}
+
+func main() {
+	now := time.Now().UTC()
+	podName := os.Getenv("POD_NAME")
```

Comment:
Does this work outside of k8s, for example used in binary not pod.

Reply:
If you set the environment variable in advance, that's all right.

Reply:
If you set the environment variable in advance, this is OK. The reason for passing in the POD name is the name of the folder that needs to be generated by POD name. This will restore the corresponding folder.